jwql/jwql_monitors/monitor_filesystem.py

```diff
@@ -688,8 +700,14 @@ def update_database(general_results_dict, instrument_results_dict, central_stora
             new_record['filetype'] = filetype
             new_record['count'] = instrument_results_dict[instrument][filetype]['count']
             new_record['size'] = instrument_results_dict[instrument][filetype]['size']
-            engine.execute(FilesystemInstrument.__table__.insert(), new_record)
-            session.commit()
+
+            # Protect against updated enum options that have not been propagated to
+            # the table definition
+            try:
+                engine.execute(FilesystemInstrument.__table__.insert(), new_record)
+                session.commit()
+            except DataError as e:
```

Comment:
This will prevent the monitor from crashing when it tries to add a new file suffix to the database. So the monitor will complete in that case. The issue now is that we'd have to search through otherwise successful log files for mention of an error where a file suffix failed.

Also, we need to fix the database tables to add `msa` to the enum. This will have to be done manually on the servers. I think ADJUST TYPE is what we want here, but I'm not yet sure of exactly how to do it.

Reply:
@bhilbert4 I went ahead and logged into the test, dev and production databases and added `msa` to the `filetype` enum.